bootstrap

```diff
@@ -84,16 +84,30 @@ compile(App, FirstFiles) ->
     code:add_path(filename:join(Dir, "ebin")),
     FirstFilesPaths = [filename:join([Dir, "src", Module]) || Module <- FirstFiles],
     Sources = FirstFilesPaths ++ filelib:wildcard(filename:join([Dir, "src", "*.erl"])),
-    [compile:file(X, [{i, filename:join(Dir, "include")}
+    [compile_file(X, [{i, filename:join(Dir, "include")}
                      ,{outdir,  filename:join(Dir, "ebin")}
-                     ,return]) || X <- Sources].
+                     ,return | additional_defines()]) || X <- Sources].
+
+compile_file(File, Opts) ->
+    case compile:file(File, Opts) of
+        {ok, _Mod} ->
+            ok;
+        {ok, _Mod, []} ->
+            ok;
+        {ok, _Mod, Ws} ->
+            io:format("Warning: ~s~n", [format_warnings(File, Ws)]);
```

Comment:
`warning_as_errors` isn't there but I realized I should include that anyway and fixed a warning during that initial compile phase.